packages/legend-art/style/base/_panel.scss

```diff
@@ -226,6 +226,35 @@
     justify-content: flex-end;
   }
 
+  &__row {
+    width: 100%;
+    display: flex;
+
+    &__item {
+      min-width: 60%;
+      max-width: 80%;
+      justify-content: right;
+    }
+
+    &__actions {
+      width: 20%;
+      display: flex;
+    }
+
+    &-btn {
+      @include flexCenter;
+
+      flex: 2.8rem 0 0;
+      // padding: 0 1rem;
```

Comment:
if you don't need it, let's remove it. Also, for now we can do `width: 2.8rem` instead